google/cloud/bigtable_admin_v2/gapic/bigtable_instance_admin_client.py

```diff
@@ -1599,7 +1599,7 @@ def update_app_profile(
     def delete_app_profile(
         self,
         name,
-        ignore_warnings=None,
+        ignore_warnings,
```

Comment:
This is a breaking change. @kolea2 do we expect this from the backend? It would seem in most languages this would result in a breaking change in clients.